src/screens/CartScreen/CartScreen.tsx

```diff
@@ -0,0 +1,15 @@
+
+const CartScreen: React.FC = () => {
+    return (
+        <div>
+            <header>
+                <div>
+                    <h1>Hello Word</h1>
+                </div>
+            </header>
+        </div>
```

Comment:
Sem necessidade desse Header, você pcolocou header e 2 divs, código inútil, o certo era colocar uma div e o h1, apenas.
```suggestion
        <div>
               <h1>Hello Word</h1>
        </div>
```